packages/clerk-js/src/ui/contexts/components/SignIn.ts

```diff
@@ -37,7 +37,8 @@ export const useSignInContext = (): SignInContextType => {
     throw new Error(`Clerk: useSignInContext called outside of the mounted SignIn component.`);
   }
 
-  const { componentName, ...ctx } = context;
+  const { componentName, ..._ctx } = context;
+  const ctx = _ctx.__experimental?.combinedProps || _ctx;
```

Comment:
Should these lines be reversed?
```suggestion
  const ctx = _ctx.__experimental?.combinedProps || _ctx;
  const { componentName, ..._ctx } = context;
```

Reply:
I don't think so, `componentName` only lives on the `SignInCtx` so we pull that off from the original `context` and the use the rest of the props as the context values below based on the usage of `combinedProps`.

Reply:
Ahh I missed the `_` 😅